packages/mdc-elevation/README.md

```diff
@@ -51,6 +51,21 @@ However, you can also apply elevation to specific components using `mdc-elevatio
   </div>
 ```
 
+#### Elevation Overlay
+
+The elevation overlay should appear *above* the component container but *below* the ripple. To accomplish this, the `.mdc-elevation-overlay` element should appear *before* the `.mdc-<component>__ripple` element in the DOM.  Here's sample markup for a button with a touch target.
```

Comment:
Explicitly mention above / below are referring to stacking the layer (z-index) since we're referring DOM order in following section.

Reply:
Done